charts/plot-navigator/values.yaml

```diff
@@ -0,0 +1,19 @@
+hostname: ""
```

Comment:
Why would `hostname` be different from `ingress.host`?

Reply:
Agreed, merged into a single parameter now.